src/schema/rules/files/deriv/imaging.yaml

```
@@ -93,6 +93,12 @@ anat_parametric_discrete_segmentation:
     resolution: optional
     density: optional
     description: optional
+  extensions:
+    - .nii.gz
+    - .nii
+    - .json
+    - .tsv
```

If the goal is to extend the extensions with `.tsv`, do you want to reference the existing list of extensions instead?

```suggestion
  extensions:
    - $ref: rules.files.raw.anat.parametric.extensions
    - .tsv
```

---

I do, had forgotten if it worked with extensions, thank you.

---

dang looks like dereferencing might be failing on lists